src/components/services/Services.tsx

```diff
@@ -128,49 +127,16 @@ export function Services(props: Props) {
     'Services 2'
   )
 
-  // Subscribe to Android Power Saver state, and show a warning only if it
-  // changes from off to on:
+  // Show a warning only if Android low power mode is enabled
   useAsyncEffect(
     async () => {
-      // This method is only available for Android
-      if (powerSavingOn == null) return
-
-      let airshipBridge: AirshipBridge<void> | undefined
-      const handlePowerSavingModeChanged = async (isPowerSavingModeOn: boolean) => {
-        if (isPowerSavingModeOn && airshipBridge == null) {
-          await Airship.show(bridge => {
-            airshipBridge = bridge // Capture the bridge here
-            return <AlertDropdown bridge={bridge} message={lstrings.warning_battery_saver} warning persistent />
-          }).then(() => {
-            airshipBridge = undefined
-          })
-        } else if (!isPowerSavingModeOn && airshipBridge != null) {
-          // Dismiss the alert when power-saving mode turns off and there's an
-          // active warning that wasn't dismissed
-          airshipBridge.resolve()
-        }
-      }
-
-      // Show warning if Power Saver mode is on initially on app boot
-      await handlePowerSavingModeChanged(await powerSavingOn())
-
-      // Subscribe to Power Saver mode changes
-      let subscription: EmitterSubscription | undefined
-      if (powerSavingModeChanged != null) {
-        subscription = powerSavingModeChanged(handlePowerSavingModeChanged)
-      }
-
-      // Cleanup
-      return () => {
-        if (subscription != null) {
-          subscription.remove()
-        }
-        if (airshipBridge != null) {
-          airshipBridge.resolve()
-        }
+      if (powerState.lowPowerMode) {
+        Airship.show(bridge => {
+          return <AlertDropdown bridge={bridge} message={lstrings.warning_battery_saver} warning persistent />
+        }).catch(e => console.error('Services 3 AlertDropdown', e))
```

Comment:
What is up with this `console.error`? We either use `showDevError`, or we ignore errors cleanly.